validation/views.py

```diff
@@ -153,4 +163,55 @@ def add_cors_headers(response):
     return response
 
 
+def login(request):
+    """
+    Serves the login page and sets cookies on successful login
+    """
+
+    if request.method == "GET":
```

Comment:
Login should use the `POST` method. Among other things, this would record user’s passwords in the server log. Other than that it could be ok, but you may want to use the built-in [LoginView](https://docs.djangoproject.com/en/3.1/topics/auth/default/#django.contrib.auth.views.LoginView)

Reply:
Excellent suggestion! I ideally don't want passwords being stored in plain text, so I'll look into the LoginView.